pkgs/development/compilers/avian/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, zlib, jdk }:
+
+stdenv.mkDerivation {
+  name = "avian-1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "readytalk";
+    repo = "avian";
+    rev = "72c0547f4bdedc1ba8da2882317e3f758deab23f";
```

Comment:
To elaborate: `mkDerivation` is a single-parameter function that maps an attribute set to a derivation. The `rec` keyword pertains to its argument.